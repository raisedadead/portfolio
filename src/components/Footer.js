import React from 'react';
import SocialNav from './Nav/SocialNav';
import { Box, Text } from 'mineral-ui';

const Footer = () => (
  <footer>
    <Box>
      <SocialNav />
      <Text as="h6" align="center">
        {` `}
        &copy; {new Date().getFullYear()} Mrugesh Mohapatra.
      </Text>
    </Box>
  </footer>
);

export default Footer;
