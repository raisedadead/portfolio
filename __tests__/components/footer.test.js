import React from 'react';
import { render } from 'react-testing-library';

import Footer from '../../src/components/Footer';

describe(`Footer`, () => {
  it(`renders correctly -- according to react-test-renderer`, () => {
    const {
      container: { firstChild }
    } = render(<Footer />);
    expect(firstChild).toMatchSnapshot();
  });
});
