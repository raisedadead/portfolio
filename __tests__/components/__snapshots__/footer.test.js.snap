// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly -- according to react-test-renderer 1`] = `
<footer>
  <section
    class="css-1big75 elghiqq0"
  >
    <nav
      class="css-o92m70-PrimaryNav e1befqft0"
    >
      <a
        aria-label="Twitter"
        class="e1befqft1 css-1fszmt7 e70ypka2"
        data-index="0"
        href="https://twitter.com/raisedadead"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-pw07op e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              <i
                class="fa"
              />
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="GitHub"
        class="e1befqft1 css-1fszmt7 e70ypka2"
        data-index="1"
        href="https://github.com/raisedadead"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-pw07op e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              <i
                class="fa"
              />
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="LinkedIn"
        class="e1befqft1 css-1fszmt7 e70ypka2"
        data-index="2"
        href="https://linkedin.com/in/mrugeshm"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-pw07op e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              <i
                class="fa"
              />
            </span>
          </span>
        </span>
      </a>
    </nav>
    <h6
      class="css-pdf1eb e19le4zx0"
    >
       
      Â© 
      2019
       Mrugesh Mohapatra.
    </h6>
  </section>
</footer>
`;
